src/transport.cpp

```diff
@@ -1353,8 +1352,11 @@ namespace quicr {
     void Transport::RemoveAllTracksForConnectionClose(ConnectionContext& conn_ctx)
     {
         // clean up subscriber handlers on disconnect
-        for (const auto& [sub_id, handler] : conn_ctx.sub_tracks_by_request_id) {
+        for (const auto& [req_id, handler] : conn_ctx.sub_tracks_by_request_id) {
             RemoveSubscribeTrack(conn_ctx, *handler, false);
+            if (handler->GetConnectionId() == conn_ctx.connection_handle) {
```

Comment:
This was the root cause of the problem causing the churn of sending subscribe update (aka pause) on the same connection that was closed. It was triggered by not receiving an unsubscribe and when the publisher and subscriber for the same full track name are the same connection.